exercises/acronym/example.h

```diff
@@ -0,0 +1,12 @@
+#if !defined(ACRONYM_H)
+#define ACRONYM_H
+
+#include <string>
+
+namespace acronym {
+
+const std::string acronym(std::string const&);
```

Comment:
I'm fairly sure that returning a const value is not useful in this situation, I can't think of very many situations where it would be useful. You probably want to return just `std::string`, though if there's a reason for the const I'd love to know about it :)

Reply:
Agreed! :) Will fix this.